app/Controllers/entryController.php

```diff
@@ -169,7 +169,9 @@ public function optimizeAction(): void {
 			Minz_Request::forward($url_redirect, true);
 		}
 
-		@set_time_limit(300);
+                if (function_exists('set_time_limit')) {
+                        set_time_limit(300);
```

Comment:
Is that truly the only reason it might fail? This way it seems that you're changing behavior compared to just leaving it with `@` to say that any failure doesn't really matter.
```suggestion
                        @set_time_limit(300);
```

Reply:
@Frenzie set_time_limit only raise fatal error if it's disabled.

> Prior to PHP 8.0.0, it was possible for the @ operator to disable critical errors that will terminate script execution. For example, prepending @ to a call of a function which did not exist, by being unavailable or mistyped, would cause the script to terminate with no indication as to why.
from https://www.php.net/manual/en/language.operators.errorcontrol.php

Thus with function_exists (supported from PHP 4 to 8), @ decorator is not needed.
Any way, preserve the @ is also okay.

Reply:
preserve @ decorator commit appended